runtime/doc/api.txt

```diff
@@ -1082,7 +1082,12 @@ nvim_input_mouse({button}, {action}, {modifier}, {grid}, {row}, {col})
                     same specifiers are used as for a key press, except that
                     the "-" separator is optional, so "C-A-", "c-a" and "CA"
                     can all be used to specify Ctrl+Alt+click.
-      • {grid}      Grid number if the client uses |ui-multigrid|, else 0.
+      • {grid}      Grid number if the client uses |ui-multigrid|, with manual
+                    positioning of the windows else 0. If you are using
+                    multigrid and let nvim determine the positioning of the
+                    windows when using multigrid, you can pass 0 and nvim will
+                    find the window grid for you. You can combine both
```

Comment:
should this doc call out : 

> but as long as `compindex`, `screen_row` and `screen_col` is used from the `win_float_pos` UI event, it will work correctly.

And perhaps also add that callout to `:help dev-ui`. 

(We need a more complete "UI dev guide", but until we have that, collecting these notes somewhere is a good starting point.)

Reply:
I will look into this tomorrow

Reply:
More detailed documentation has been added to `develop.txt`